exercises/robot-name/robot-name.jl

```diff
@@ -0,0 +1,11 @@
+type Robot
```

Comment:
This exercise is a bit more advanced, so I think providing less help in the skeleton would make sense.

This is one of the first exercises that introduces types though, so some guiding towards them is probably a good idea. Perhaps just having the function skeleton for `reset(instance::Robot)` is enough of a hint that the user needs to implement his own type?